beets/util/mb_tagger_server.py

```diff
@@ -0,0 +1,22 @@
+import socket
```

Comment:
You should add a copyright blurb, encoding info, and docstring to the file as well. (See other files.)